src/platform/windows/audio.cpp

```diff
@@ -793,6 +802,26 @@ namespace platf::audio {
         }
       }
 
+      // When switching to a Steam virtual speaker device, try to retain the bit depth of the
+      // default audio device. Switching from a 16-bit device to a 24-bit one has been known to
+      // cause glitches for some users.
+      int wanted_bits_per_sample = 32;
+      auto current_default_dev = default_device(device_enum);
+      if (current_default_dev) {
+        audio::prop_t prop;
+        prop_var_t current_device_format;
+
+        // clang-format off: easier to read this split over 2 lines
```

Comment:
I know it's not ideal to have this on a long line, but I'd prefer not to have clang-format exceptions for this. I'm open to different clang-format rules though to help make things more readable.